python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -0,0 +1,286 @@
+
+        self.rfs = {worker: c.submit(RandomForestRegressor._func_build_rf,
+                             n, self.n_estimators_per_worker[n],
+                             max_depth,
+                             handle, max_features, n_bins,
+                             split_algo, split_criterion,
+                             bootstrap, bootstrap_features,
+                             verbose, min_rows_per_node,
+                             rows_sample, max_leaves,
+                             accuracy_metric, quantile_per_tree,
+                             random.random(),
+                             workers=[worker])
+            for worker, n in ws}
+
+        rfs_wait = list()
+        for r in self.rfs.values():
+            rfs_wait.append(r)
+
+        wait(rfs_wait)
+
+    @staticmethod
+    def _func_build_rf(n, n_estimators, max_depth,
+                       handle, max_features, n_bins,
+                       split_algo, split_criterion,
+                       bootstrap, bootstrap_features,
+                       verbose, min_rows_per_node,
+                       rows_sample, max_leaves,
+                       accuracy_metric, quantile_per_tree,
+                       r):
+
+        return cuRFR(n_estimators=n_estimators, max_depth=max_depth, handle=handle,
+                     max_features=max_features, n_bins=n_bins,
+                     split_algo=split_algo, split_criterion=split_criterion,
+                     bootstrap=bootstrap, bootstrap_features=bootstrap_features,
+                     verbose=verbose, min_rows_per_node=min_rows_per_node,
+                     rows_sample=rows_sample, max_leaves=max_leaves,
+                     accuracy_metric=accuracy_metric,
+                     quantile_per_tree=quantile_per_tree)
+
+    @staticmethod
+    def _fit(model, X_df, y_df, r):
+        return model.fit(X_df, y_df)
+
+    @staticmethod
+    def _predict(model, X, r):
+        return model.predict(X)
+
+    def fit(self, X, y):
+        """
+        Fit the input data to Random Forest classifier
+
+        Parameters
+        ----------
+        X : dask_cudf.Dataframe containing dense matrix (floats or doubles) of shape (n_samples, n_features).
+        y : dask_cudf.Dataframe containing dense matrix (floats or doubles) of shape (n_samples, 1)
+        """
+        c = default_client()
+
+        X_futures = c.sync(extract_ddf_partitions, X)
+        y_futures = dict(c.sync(extract_ddf_partitions, y))
+
+        f = list()
+        for w, xc in X_futures:
+            f.append(c.submit(RandomForestRegressor._fit, self.rfs[w], xc, y_futures[w], random.random(),
+                             workers=[w]))
+
+        wait(f)
+
+        return self
+
+    def predict(self, X):
+        """
+        Predicts the labels for X.
+
+        Parameters
+        ----------
+        X : dask_cudf.Dataframe containing dense matrix (floats or doubles) of shape (n_samples, n_features).
```

Comment:
I apologize for being so late to discover this, however, after following through the predict code a little more closely, I notice you are using a numpy array in the pytest for predict. It says here it accepts a dataframe, but I'm not sure a dask cudf dataframe would work as input here, since the call to `client.submit` would be sending the dataframe object to the workers (which would need to have `compute()` called on it in order to become a dataframe local to that worker, so that it could be passed into the predict() of the underlying SG RF instance).